### ISSUE TITLE
OverflowScore calculation method

### ISSUE THREAD

**Issue by jack-gracie** [liangrj2014/ISPD25_contest#14]
Hi, I was wondering if the overflowScore calculation method is identical to that in ISPD24. Additionally, I would appreciate it if you could share when the manually-defined weights w1-4 will be announced.
![图片](https://github.com/user-attachments/assets/6585c96b-269b-4062-8ff5-d3afb6c0ad65)

**Comment by liangrj2014** [comment]
Yes, the overflowScore calculation method is identical to that in ISPD24. We will release the weights soon
